docs/maintainers/releasing.md

```diff
@@ -1,77 +1,74 @@
-* Update [CHANGELOG.md](../../CHANGELOG.md) with the major change list.
-* Create an [annotated tag][tag] for the release, with the major change list as the message.
-* Push the release tag, and create a release draft with the major change list.
-* Build packages and add them to the release draft.
-* Create NuGet packages and publish them to [powershell-core feed][ps-core-feed].
-* Publish the release draft.
-* Update documentation and scripts to use the links to new packages.
-* Push a branch named `docker` to `powershell/powershell` repository to trigger building docker images.
-* Delete the `docker` branch once the builds are successful at [powershell docker hub](https://hub.docker.com/r/microsoft/powershell/builds/).
-
-Preparing
-=========
-
-PowerShell releases use [Semantic Versioning][semver].
-Until we hit 6.0, each sprint results in a bump to the build number,
-so `v6.0.0-alpha.7` goes to `v6.0.0-alpha.8`.
-
-When a particular commit is chosen as a release,
-we create an [annotated tag][tag] that names the release,
-and list the major changes since the previous release.
-An annotated tag has a message (like a commit),
-and is *not* the same as a lightweight tag.
-Create one with `git tag -a v6.0.0-alpha.7`.
-Our convention is to prepend the `v` to the semantic version.
-The summary (first line) of the annotated tag message should be the full release title,
-e.g. 'v6.0.0-alpha.7 release of PowerShell'.
-
-While creating a release, it is advised to make a new branch such that
-necessary documentation updates and hot fixes can be made,
-without having to include all changes made to master.
-This release branch can be reviewed by the normal PR process.
-
-When the annotated tag is finalized, push it with `git push --tags`.
-GitHub will see the tag and present it as an option when creating a new [release][].
-Start the release, use the annotated tag's summary as the title,
-and save the release as a draft while you upload the binary packages.
-
-Just as important as creating the release is updating the links on our readme,
-and the package names in the installation instructions.
-The AppVeyor build number should also be incremented.
-
-When creating the packages, please ensure that the file path does not contain user names.
+# PowerShell Core Releasing Process
+
+## Release Steps
+
+1. Create a branch named `release` in `PowerShell/PowerShell` repository. All release related changes should happen in this branch.
+1. Prepare packages
+   - [Build release packages](#building-packages)
+   - Sign the MSI packages and DEB/RPM packages.
+   - Install and verify the packages. (_could be automated [Issue #3942][]_)
+1. Update documentation, scripts and Dockerfiles
+   - Summarize the change log for the release. It should be reviewed by PM(s) to make it more user-friendly.
+   - Update [CHANGELOG.md](../../CHANGELOG.md) with the finalized change log draft.
+   - Update other documents and scripts to use the new package names and links.
```

Comment:
have any of us published a script to do this? I created a few one-liners a while ago to do it and I think we all have different scripts. A single unified script would be nice.